index.html.in

```diff
@@ -217,22 +217,22 @@
                                   </div>
                               </div>
 
-                              <div class="modal-backdrop" id="fileDropBackdrop" style="display: none; z-index: 1000;">
-                                <div class="well well-lg" style="text-align: center; position: absolute; top: 35%; left: 35%;">
-                                    <h1 style="font-weight: bold;" data-text="Drop_Document">Drop your document</h1>
+                              <div class="modal-backdrop" id="fileDropBackdrop">
+                                <div class="well well-lg" id="modalWell">
+                                    <h1 id="dropDocHeader" data-text="Drop_Document">Drop your document</h1>
```

Comment:
Instead of giving all three elements here unique `id`s, its much more preferable to use `. fileDropBackdrop h1`, `. fileDropBackdrop .well`, etc. This same comment might apply in a couple of other places.